build.gradle

```diff
@@ -1041,6 +1039,7 @@ project(':iceberg-spark3-runtime') {
     relocate 'com.google', 'org.apache.iceberg.shaded.com.google'
     relocate 'com.fasterxml', 'org.apache.iceberg.shaded.com.fasterxml'
     relocate 'com.github.benmanes', 'org.apache.iceberg.shaded.com.github.benmanes'
+    relocate 'org.antlr.v4.runtime', 'org.apache.iceberg.shaded.org.antlr.v4.runtime'
```

Comment:
I guess we don't bundle antlr in `iceberg-spark-runtime` hence no change there.

Reply:
Yes, this change only affect `iceberg-spark3-runtime`